src/alg_utils.jl

```diff
@@ -76,6 +76,10 @@ isfsal(alg::SSPRK932) = false
 isfsal(alg::SSPRK54) = false
 isfsal(alg::SSPRK104) = false
 
+isfals(alg::PSRK3p5q4) = false
+isfals(alg::PSRK3p6q5) = false
+isfals(alg::PSRK4p7q6) = false
```

Comment:
```suggestion
isfsal(alg::PSRK3p5q4) = false
isfsal(alg::PSRK3p6q5) = false
isfsal(alg::PSRK4p7q6) = false
```

Can we please get some tests for this?